src/schemas/json/pantsbuild-2.14.0.json

```
@@ -8,453 +8,453 @@
       "properties": {
         "backend_packages": {
           "default": [],
-          "description": "Register functionality from these backends.\n\nThe backend packages must be present on the PYTHONPATH, typically because they are in the Pants core dist, in a plugin dist, or available as sources in the repo.",
+          "description": "Register functionality from these backends.\n\nThe backend packages must be present on the PYTHONPATH, typically because they are in the Pants core dist, in a plugin dist, or available as sources in the repo.\n\nhttps://www.pantsbuild.org/v2.14/docs/reference-GLOBAL#backend_packages",
```

```suggestion
          "description": "Register functionality from these backends\nhttps://www.pantsbuild.org/v2.14/docs/reference-GLOBAL#backend_packages",
```

Required fix: broken line delimiters: one consecutive `\n` expected.
Required fix: broken description: too long description.
Note: try automatically remove all sentences except the one first.

---

Thank you, this is now fixed. A description has only one sentence following a URL on the new line.